src/index.js

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import { legacy_createStore as createStore } from 'redux';
+import { legacy_createStore as createStore, applyMiddleware } from 'redux';
 import { Provider } from 'react-redux';
+import thunkMiddleware from 'redux-thunk';
+import { composeWithDevTools } from 'redux-devtools-extension';
 import './index.css';
 import { BrowserRouter as Router } from 'react-router-dom';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 import rootReducer from './redux/configureStore';
 
-const store = createStore(rootReducer);
+const composedEnhancer = composeWithDevTools(applyMiddleware(thunkMiddleware));
+
+const store = createStore(rootReducer, composedEnhancer);
```

Comment:
- [ ] Please Move everything that deals with the redux store to the `redux/configeStore.js` file for best practices and using the `ducks` structure. you can move lines 3, 5, 6, 13, and 15. and then you can only import store in the `index.js` file.

[Hint] You can import `configureStore` instead of old `createStore`.